### PR TITLE
JIRA-MEN-50: Client+Server TLS authenticated -bootstrap command. TODO finalise.

### PR DESCRIPTION
**Pull request by jimis**
DO NOT MERGE.

### CODE REVIEW

File: bootstrap.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	fmt.Println("Sending HTTP GET to: ", serverURL)
+
+	response, err := httpClient.Get(serverURL)
+	if err != nil {
+		fmt.Println("HTTP GET failed:", err)
+		return nil // TODO
+	}
+
+	fmt.Println("Received headers:", response.Header)
+	respData, err := ioutil.ReadAll(response.Body)
```
Comment (kacf) [inline bootstrap.go]: I think you need to do `response.Body.Close()` afterwards.

Comment (kacf) [inline bootstrap.go]: A typical use case for `defer`. :-)

Comment (jimis) [inline bootstrap.go]: So no destructors for go. :-p Thanks, review much appreciated!

Comment (kacf) [inline bootstrap.go]: No unfortunately not, it's one of the things I still miss from C++. :-)

**Comment by pasinskim** [comment]
This is already added to master after some refactoring. I will close this one.

**Pull request closed without merging**